packages/api/cms-api/src/access-log/access-log.interceptor.ts

```diff
@@ -0,0 +1,59 @@
+import { CallHandler, ExecutionContext, Injectable, Logger, NestInterceptor } from "@nestjs/common";
+import { GqlExecutionContext } from "@nestjs/graphql";
+
+/** TODOs
+ * Docs are missing
+ * Api password check
+ */
+const DAM_IMAGE_PATH = "/dam/images/:hash/:fileId";
+const DAM_FILE_PATH = "/dam/files/:hash/:fileId";
```

Comment:
We also have  the preview URLs, which are used in the Admin previews. Does the logger ignore them too? Should it ignore them?

Reply:
thanks, i will check

Reply:
preview urls will not be ignored actually. Should we ignore them too?